scripts/docker_version_control.sh

```diff
@@ -19,12 +19,11 @@ image_tag=`grep "^FROM " $DOCKERFILE | awk '{print$2}'`
 image=`echo $image_tag | cut -f1 -d:`
 tag=`echo $image_tag | cut -f2 -d:`
 
-# if docker image not in white list, exit
-if [[ "$IMAGENAME" != sonic-slave-* ]] && [[ "$IMAGENAME" != docker-base* ]];then
-    exit 0
-fi
-
 if [[ ",$SONIC_VERSION_CONTROL_COMPONENTS," == *,all,* ]] || [[ ",$SONIC_VERSION_CONTROL_COMPONENTS," == *,docker,* ]]; then
+    # if docker image not in white list, exit
+    if [[ "$IMAGENAME" != sonic-slave-* ]] && [[ "$IMAGENAME" != docker-base* ]] && [[ "$IMAGENAME" != debian* ]] && [[ "$IMAGENAME" != multiarch/debian-debootstrap* ]];then
```

Comment:
>* [](http://example.com/codeflow?start=109&length=1)

Do we really need the wildcard?

Reply:
there are sonic-slave-buster, sonic-slave-stretch, debian:buster, debian:stretch, and so on.

Reply:
`debian*` can be improved as `debian:*` ?